packages/runtime/runtime-definitions/src/dataStoreContext.ts

```diff
@@ -82,11 +82,6 @@ export interface IContainerRuntimeBase extends
      */
     setFlushMode(mode: FlushMode): void;
 
-    /**
-     * Executes a request against the container runtime
-     */
-    request(request: IRequest): Promise<IResponse>;
```

Comment:
ContainerRuntimeBase no longer has request, but it does have getEntryPoint which still isn't ideal. ideally the datastore is give handles to any dependencies at construction time which the data store persists, and it never need to reach back up